### ISSUE TITLE
Very strange errors

### ISSUE THREAD

**Issue by mcblum** [tymondesigns/jwt-auth#765]
After using this package on many, many apps for years, something is happening that I can't explain or figure out! 

When I log in the attempt is sucessful and it triggers the JSON response, but it errors now every time.

````
InvalidArgumentException in JsonResponse.php line 58: Syntax error
````

The above error actually randomly changes, too.

````
at JsonResponse->setData(array('token' => array('token' => 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOjEsImlzcyI6Imh0dHA6XC9cL2FwaS5zaWduYXR1cmVhcHByYWlzYWxzLmNvbS5uYXRpb25hbG1hY2hpbmUuaW9cL2F1dGhlbnRpY2F0ZSIsImlhdCI6MTQ2NjE4NTg2NCwiZXhwIjoxNDY2MTg5NDY0LCJuYmYiOjE0NjYxODU4NjQsImp0aSI6ImJjZjZjMzA2NmFmYjQxMTdiNDMzNTY1NWUzMTVlNmY5In0.Ybt0JRgW3zt_kQiMFPK68AmdS56e_AyvTL3CkqNLhnE'), 'user' => object(User), 'message' => 'You have successfully logged in!')) in JsonResponse.php line 49
````

I have been able to fix the error by doing `php artisan migrate:refresh' which to me makes to sense at all. 

Thanks for your work developing this package, and please let me know if you have any ideas!

**Comment by mcblum** [comment]
Never mind! Feel freeze to close. It was actually an error within the JSON within the user object and nothing to do with JWT. 

Thanks again for this package and I sent over a beer!

**Comment by tymondesigns** [comment]
Glad you sorted it. Thanks for the beer! 👍🍻

**Comment by chebyte** [comment]
@mcblum  please dont forget to approval your wish on beerpay, so @tymondesigns can buy his beer! 
@mcblum  this wish is waiting for owner approval!

**Comment by mcblum** [comment]
@chebyte wait what? I already sent that!

**Comment by tymondesigns** [comment]
@mcblum I think it's because you sent via a "wish" which requires you to say "I'm happy that this wish has been fulfilled" as apposed to a straight up "support donation"

Thanks again

**Comment by mcblum** [comment]
One would think if I sent a 'wish' and the issue was resolved, that would be all you need to do. I'd like to open an issue with that service :)

**Comment by chebyte** [comment]
@mcblum exactly, as @mcblum said, you need to check that the issue was fixed properly, then you have to accept their work, and the developer will get the donation!.
